music21/roman.py

```diff
@@ -748,83 +690,66 @@ def romanNumeralFromChord(chordObj,
     ...     )
     <music21.roman.RomanNumeral Sw43 in g minor>
 
-
-    With correct key context:
+    Naturally, the user can still specify a different key:
 
     >>> roman.romanNumeralFromChord(
     ...     chord.Chord('E-4 G4 C#5'),
     ...     key.Key('G')
     ...     )
     <music21.roman.RomanNumeral It6 in G major>
 
-    With incorrect key context does not find an augmented 6th chord:
+    And if that key makes the chord no longer an augmented 6th chord
+    then the returned RomanNumeral will reflect this:
 
     >>> roman.romanNumeralFromChord(
     ...     chord.Chord('E-4 G4 C#5'),
     ...     key.Key('C')
     ...     )
     <music21.roman.RomanNumeral #io6b3 in C major>
 
+    As always, the case of sixth and seventh degrees in minor pose
+    additional problems given the range of widely adopted conventions.
+    Music21 generally handles this with the :class:`music21.roman.Minor67Default`
+    as explained in the :class:`music21.roman.romanNumeral` and especially the
+    :meth:`~music21.roman.RomanNumeral.adjustMinorVIandVIIByQuality`.
```

Comment:
Just FYI if #657 gets merged first we should strike the reference to this method and only point folks to `Minor67Default`.

Reply:
This is too complex for a basic user.